NEWS.md

```diff
@@ -1,10 +1,19 @@
+# epidatr 1.2.0
+
+## Changes
+
+- Improve handling of the `EPIDATR_USE_CACHE` environment variable, allowing it
+  to be any value convertable by `as.logical()` and handle the case when it
+  can't be converted.
```

Comment:
nit: please include PR number here

```suggestion
  can't be converted (#273).
```

Reply:
ah my bad, I set it up to auto-merge, let's add this in a next PR